scarlet/observation.py

```diff
@@ -219,8 +232,8 @@ def match(self, model_frame):
     def _convolve_band(self, model, diff_kernel_fft):
```

Comment:
The method should be renamed to `_convolve`, and since `diff_kernel_fft` is an attribute of self, we can drop it from the argument list. This affects line 254.